src/app/components/MediaLoader/utils/buildPlaceholderConfig/index.ts

```diff
@@ -0,0 +1,56 @@
+import moment from 'moment-timezone';
+import formatDuration from '#app/lib/utilities/formatDuration';
+import { Translations } from '#app/models/types/translations';
+import { getPlaceholderSrcSet } from '#app/lib/utilities/srcSet';
+import { PlaceholderConfig, MediaType } from '../../types';
+
+export default ({
+  title,
+  duration,
+  durationISO8601,
+  holdingImageURL,
+  placeholderImageOriginCode,
+  placeholderImageLocator,
+  translations,
+  guidanceMessage,
+  type,
+}: {
+  title: string;
+  duration: number;
+  holdingImageURL: string;
+  placeholderImageOriginCode: string;
+  placeholderImageLocator: string;
+  type: MediaType;
+  durationISO8601?: string;
+  translations?: Translations;
+  guidanceMessage?: string | null;
```

Comment:
Looks good 👍 

Should we move this into a type just above the function? Not really sure if there is a design pattern, but I usually find it easier to read/update if multi-line types are broken out into their own `type XXX { }`

Reply:
Ah yes, can do that - that's the pattern we usually follow 👍